README.md

```diff
@@ -1,3 +1,51 @@
+# OLMo On The Kempner Institute HPC Cluster
+This section has been added to OLMo `README` page for Kempner Community to help them with a step by step guideline to run OLMo on the HPC cluster and start exploring.
+## 1. Installation on The HPC Cluster 
+### 1.1. Create Conda Environment and Activate it
+First create your conda environment using the following command:
+```bash
+conda create --prefix [path_to_your_env] python=3.10
+```
+Activate the environment using:
+```bash
+conda activate [path_to_your_env]
+```
+### 1.2. Install PyTorch
+First install [PyTorch](https://pytorch.org) according to the instructions specific to your operating system.
+```bash
+pip3 install torch
+```
+### 1.3. Install OLMo from Source Code 
+To install from source (recommended for training/fine-tuning) run:
+
+```bash
+git clone https://github.com/KempnerInstitute/OLMo.git
+cd OLMo
+pip install -e .[all]
+```
+## 2. Run OLMo on the Kempner Institute HPC Cluster
+Now that we have the Conda environment ready, it's time to run OLMo. To do that, we need a config file to pass it to the training script to specify all OLMo configs and hyperparameters as well as a slurm script to submit the job on the HPC cluster. 
+
+### 2.1. Config file
+ A config file has been provided in [configs/kempner_dev/7b_Olmo.yaml](configs/kempner_dev/7b_Olmo.yaml) which enables running of a 7b OLMo model training on 4 GPUs using the `c4` data which is tokenized by `t5-base` tokenizer. You can take this config file and may adjust its different hyperparameters based on your need.
```

Comment:
This line says that we are training 7B using 4 gpus, is this efficient for general use? Furthermore, should we consider adding a 1B config file (in addition to the separate ddp and fsdp configs), so that we end up with four configs: 1B_DDP, 1B_FSDP, 7B_DDP, 7B_FSDP?

Reply:
We need one general config file that contains all the common configs and then have two small config file to contains only for FSDP and DDP part.
When running OLMo with the common config file + fsdp config file it runs it using fsdp and to switch to DDP we use common config file + ddp config file.

Reply:
Thats fine, but my point is that the reason behind doing all of this is to make it easy for people to run out of the box for their own experiments. Our default config file and slurm script tries to use 4 gpus to run a 7B model with DDP (impossible) and FSDP (inefficient with 4 gpus). My question is whether we should add the 1B config file and have it be the default in the readme instead.

Reply:
I got the point; yes that would be fine. definitely, for DDP it should be smaller size.

Reply:
ended up having 1b config for ddp and 7b config for fsdp